test/test-jsdomparser.js

```diff
@@ -6,7 +6,7 @@ var readability = require("../index.js");
 var JSDOMParser = readability.JSDOMParser;
 
 var BASETESTCASE = '<html><body><p>Some text and <a class="someclass" href="#">a link</a></p>' +
-                   '<div id="foo">With a <script>With < fancy " characters in it because' +
+                   '<div id="foo">With a <script>With &lt; fancy " characters in it because' +
```

Comment:
That's not what I said... why is it *necessary* to replace them at all? Do the tests break if you don't replace them?